### DISCUSSION TITLE
大佬问一下，这个地方显示无法购买，到时候18.00这个配置要不要重新获取

### DISCUSSION THREAD

**Discussion by LingFengJUE** [mikumifa/biliTickerBuy#247] [提问专区]
![微信截图_20240629170041](https://github.com/mikumifa/biliTickerBuy/assets/79570784/7cc1debe-7788-4b02-b4a6-d8665271424b)

大佬问一下，这个地方显示无法购买，到时候18.00这个配置要不要重新获取

**Comment by mikumifa** [comment]
不用

**Reply by LingFengJUE** [reply]
好的，谢谢大佬
